lib/membrane_rtc_engine/engine.ex

```diff
@@ -1135,6 +1136,16 @@ defmodule Membrane.RTC.Engine do
   end
 
   defp setup_endpoint(endpoint_entry, opts, state) do
+    with %Endpoint.WebRTC{} = endpoint_entry do
+      Membrane.TelemetryMetrics.conditional_execute(
+        fn -> if(opts[:peer_metadata], do: true, else: false) end,
```

Comment:
I think this shouldn't be conditional. There should always be some default value.
We should also do this for `update_peer_metadata` event